src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -735,6 +775,12 @@ class DistributedClosestPointImpl
     auto it = bvh->getTraverser();
     const int rank = m_rank;
 
+    double* sqDistThresh =
+      axom::allocate<double>(1, axom::execution_space<ExecSpace>::allocatorID());
```

Comment:
Could we use`axom::Array` here instead?

Reply:
That might give the wrong impression -- since it is a single scalar value for entire kernel.